packages/paste-website/src/components/open-graph-card/AssetStatus.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
```

Comment:
These files seem like pretty close duplications of the status components Rich created in the [feat(website): add component overview information from airtable](https://github.com/twilio-labs/paste/pull/1529/files#) PR. Is there a reason they're being recreated here? (duped file: `packages/paste-website/src/components/component-status/AssetStatus.tsx`)

Reply:
I wanted to make the icons and text bigger and couldn't be bother to make the existing one do that? 🤷🏼

Reply:
iiiiiiinteresting... 🤔